tests/functional/conftest.py

```diff
@@ -19,6 +19,7 @@
     "es.index": ELASTICSEARCH_INDEX,
     "h.app_url": "http://example.com",
     "h.authority": "example.com",
+    "h.sentry_dsn_frontend": "FAKE_SENTRY_DSN",
```

Comment:
JavaScript code is not executed during backend functional tests so this won't cause it to send requests to Sentry. And this makes the functional tests more similar to prod, by making it render the Sentry config which (in the past) could have resulted in leaked DB connections